faster_sam/cloudformation.py

```diff
@@ -266,12 +248,12 @@ def find_nodes(self, tree: Dict[str, Any], node_type: NodeType) -> Dict[str, Any
 
     def lambda_handler(self, resource_id: str) -> str:
         """
-        Constructs the Lambda handler path for a given resource ID.
+        Constructs the Lambda handler path from the given resource id.
```

Comment:
Outra sugestão:

_Returns a string representing the full module path for a Lambda Function handler._

_The path is built by joining the code URI and the handler attributes on the CloudFormation for the given Lambda Function identified by resource_id._